types/emscripten/emscripten-tests.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="emscripten" />
```

Comment:
Don't you want to import emscripten here instead?

Reply:
I believe Import statement does not make sense much in Emscripten's context.
Emscripten itself not a module nor a library actually. It is a C/C++ => WebAssembly compiler toolchain. It generates WebAssembly (.wasm) + a customizable wrapper JS (this is not a ES6 module) code + optional HTML file as output.

@type/emscripten is to give the wrapper JS code typing definitions.